packages/popover/src/vaadin-popover-overlay.js

```diff
@@ -30,6 +30,10 @@ class PopoverOverlay extends PopoverOverlayMixin(DirMixin(ThemableMixin(PolylitM
     return [
       overlayStyles,
       css`
+        :host([modeless][with-backdrop]) [part='backdrop'] {
+          pointer-events: none;
```

Comment:
Why should it be different from other overlays? I mean shouldn't other overlays have it too?

Reply:
As discussed internally with @rolfsmeds, the idea is to allow using backdrop for modeless popover, which would keep interacting with background elements possible:

> A separate withBackdrop is actually nice since that would allow you to have a curtain to visually emphasize the popover but still allow interaction with stuff below (e.g. target click)

Reply:
Ah, I see, other components don't separately expose withBackdrop property.